src/main/java/eu/nimble/service/delegate/Delegate.java

```diff
@@ -401,8 +442,300 @@ public Response postPartySearch(Map<String, Object> body) {
     }
     
     /***********************************   indexing-service/party/search - END   ***********************************/
-    
-    
+
+    /***********************************   Catalogue Service **************************************************/
```

Comment:
I sent a message to Oliver to check which REST calls from the catalog service are needed.
no need to implement a wrapper to a rest call if it's not used by the frontend service in the federated use case.
will update once I have an answer from him